docs/sources/administration/roles-and-permissions/access-control/rbac-provisioning.md

```diff
@@ -12,8 +12,6 @@ weight: 60
 
 You can create, change or remove [Custom roles]({{< relref "./manage-rbac-roles/#create-custom-roles-using-provisioning" >}}) and create or remove [basic role assignments]({{< relref "./assign-rbac-roles/#assign-a-fixed-role-to-a-basic-role-using-provisioning" >}}), by adding one or more YAML configuration files in the `provisioning/access-control/` directory.
 
-If you choose to use provisioning to assign and manage role, you must first enable it.
```

Comment:
This is not true anymore and creates confusion.

Reply:
Good catch!